README.md

```diff
@@ -38,45 +39,54 @@ $ helm init
 
 Add the minikube IP to /etc/hosts:
 ```
-$ sudo sh -c 'echo "$(minikube ip) dev.dispatch.vmware.com api.dev.dispatch.vmware.com" >> /etc/hosts'
+$ sudo sh -c 'echo "$(minikube ip) dev.dispatch.example.com api.dispatch.example.com" >> /etc/hosts'
 ```
 
+### Configure Authentication
+Configure an OAuth2 Client App at GitHub with instruction [How to Create OAuth Client
+App](https://vmware.github.io/dispatch/guides/create-oauth-client-app.html)
+
+You should have ``<oauth-client-id>``, ``<oauth-client-secret>`` and ``<oauth-cookie-secret>`` now.
+
+### Download Dispatch CLI
 Get the dispatch command:
 ```
 $ curl -OL https://github.com/vmware/dispatch/releases/download/v0.1.2/dispatch-darwin
 $ chmod +x dispatch-darwin
 ```
 
-Configure an OAuth2 Client App at GitHub with instruction [How to Create OAuth Client
-App](https://vmware.github.io/dispatch/guides/create-oauth-client-app.html)
-
-You should have ``<oauth-client-id>``, ``<oauth-client-secret>`` and ``<oauth-cookie-secret>`` now.
+### Configure and install Dispatch:
+Dispatch requires a Docker registry to build and push images for your functions. If you don't have a
+docker registry, you can always use Docker Hub by specifying your credentials.
 
-Configure the installation.  Substitute in your docker credentials (host and username are likely the same):
+Configure the installation by substituting docker credentials and OAuth2 client details.
 ```
 $ cat << EOF > config.yaml
 apiGateway:
-  hostname: api.dev.dispatch.vmware.com
+  hostname: api.dev.dispatch.example.com
 dispatch:
-  hostname: dev.dispatch.vmware.com
+  hostname: dev.dispatch.example.com
   image:
     host: vmware
     tag: v0.1.2
   debug: true
   openfaasRepository:
```

Comment:
can you also change this? It should be registry, not repository, and it should not be tied to openfaas - it should probably be just named `imageRegistry`. It would require a change in CLI as well, so feel free to ignore if you think it should go into separate PR (but I think we should address this quickly as this is a breaking change).

Reply:
Yeah, I thought will make it separately but yeah could do here as well not a big deal. Will push another change.

Reply:
👍